### PR TITLE
cli: remove confusing [ ] characters from ctlv3 help output

### LINKED ISSUE #10351: cli: --endpoints=[127.0.0.1:2379]
I assume `[ ]` stand for optional. `--endpoints=[127.0.0.1:2379]`   should be `--endpoints=127.0.0.1:2379` to be consistent with the rest, right? If you want to indicate that it accept a comma seperated list I would suggest to describe that in the description field?
```
OPTIONS:
      --cacert=""                               verify certificates of TLS-enabled secure servers using this CA bundle
      --cert=""                                 identify secure client using this TLS certificate file
      --command-timeout=5s                      timeout for short running command (excluding dial timeout)
      --debug[=false]                           enable client-side debug logging
      --dial-timeout=2s                         dial timeout for client connections
  -d, --discovery-srv=""                        domain name to query for SRV records describing cluster endpoints
      --endpoints=[127.0.0.1:2379]              gRPC endpoints
      --hex[=false]                             print byte strings as hex encoded strings
      --insecure-discovery[=true]               accept insecure SRV records describing cluster endpoints
      --insecure-skip-tls-verify[=false]        skip server certificate verification
      --insecure-transport[=true]               disable transport security for client connections
      --keepalive-time=2s                       keepalive time for client connections
      --keepalive-timeout=6s                    keepalive timeout for client connections
      --key=""                                  identify secure client using this TLS key file
      --user=""                                 username[:password] for authentication (prompt if password is not supplied)
  -w, --write-out="simple"                      set the output format (fields, json, protobuf, simple, table)
```

### PR DESCRIPTION
**Pull request by gertcuykens**
especially --endpoints=[127.0.0.1:2379] is confusing and inconsistent

Fixes #10351

### CODE REVIEW

File: etcdctl/ctlv3/help.go [hunk 1]
```diff
@@ -117,21 +117,17 @@ func etcdFlagUsages(flagSet *pflag.FlagSet) string {
...
 		} else {
 			format = format + "=%s"
 		}
-		if len(flag.NoOptDefVal) > 0 {
-			format = format + "]"
-		}
 		format = format + "\t%s\n"
 		shorthand := flag.Shorthand
-		fmt.Fprintf(x, format, shorthand, flag.Name, flag.DefValue, flag.Usage)
+		defValue := strings.Replace(flag.DefValue, "[", "", -1)
```
Comment (xiang90) [inline etcdctl/ctlv3/help.go]: can you ask in cobra repo about why does it do this?

Comment (gertcuykens) [reply etcdctl/ctlv3/help.go]: I will ask, I guess something like using a plain print %v statement of the value that is expected

Comment (gertcuykens) [reply etcdctl/ctlv3/help.go]: see link below

Comment (spzala) [reply etcdctl/ctlv3/help.go]: @gertcuykens hi, which link? Did you forget to paste the link :) Thanks!

Comment (gertcuykens) [reply etcdctl/ctlv3/help.go]: lower :) `flag.DefValue for string array #803` https://github.com/spf13/cobra/issues/803

Comment (spzala) [reply etcdctl/ctlv3/help.go]: :) Thanks!

Comment (spzala) [reply etcdctl/ctlv3/help.go]: Seems like they aren't actively looking at issues. I feel like not manually overriding the brackets, as usually that's a more visual mean of expressing it's a list. @xiang90  I am OK if we want to modify the text as this PR did in the ctl.go.

File: etcdctl/ctlv3/ctl.go [hunk 2]
```diff
@@ -46,7 +46,7 @@ var (
 )
 
 func init() {
-	rootCmd.PersistentFlags().StringSliceVar(&globalFlags.Endpoints, "endpoints", []string{"127.0.0.1:2379"}, "gRPC endpoints")
+	rootCmd.PersistentFlags().StringSliceVar(&globalFlags.Endpoints, "endpoints", []string{"127.0.0.1:2379"}, "gRPC comma separated endpoint list")
```
Comment (spzala) [inline etcdctl/ctlv3/ctl.go]: may be say "comma separated gPRC endpoints"?

**Pull request closed without merging**